.github/workflows/on_push.yml

```diff
@@ -1,6 +1,11 @@
 name: "Tests"
 
-on: push
+on:
+  push:
+    branches:
+      - "**"
```

Comment:
Isn't this the default, to be triggered for all branches? Do we need to specify this?

Reply:
I came across this note and also [here](https://github.community/t/dont-run-on-tag-creation/137469/3).
```
If you define only tags or only branches, the workflow won't run for events affecting the undefined Git ref.
```

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Reply:
Sounds good, thanks for the info :+1: